src/BookNavigator/book-navigator.js

```diff
@@ -289,19 +358,21 @@ export class BookNavigator extends LitElement {
       this.bookreader = e.detail.props;
       this.bookReaderLoaded = true;
       this.bookReaderCannotLoad = false;
-      this.fullscreenMgr = new BRFullscreenMgr(this.bookreader.el);
-
-      this.initializeBookSubmenus();
-      setTimeout(() => this.bookreader.resize(), 0);
-      this.brResizeObserver = new ResizeObserver((elements) => this.reactToBrResize(elements));
-      this.brResizeObserver.observe(this.mainBRContainer);
+      this.emitLoadingStatusUpdate(true);
+      this.sharedObserver?.addObserver({
```

Comment:
Q: Can you use the `updated()` lifecycle hook to observe when a `sharedObserver` is passed in to start the observation (and possibly disconnect prior observation)? I'm sure you'd have one at this point, but that would have a tighter connection between its passing and the setup.

Reply:
yes good catch, thanks